src/core/bitop.d

```diff
@@ -276,15 +291,21 @@ version (AnyX86)
  * They may be used to guarantee a write or read cycle occurs at a specified address.
  */
 
-ubyte  volatileLoad(ubyte * ptr);
-ushort volatileLoad(ushort* ptr);  /// ditto
-uint   volatileLoad(uint  * ptr);  /// ditto
-ulong  volatileLoad(ulong * ptr);  /// ditto
+pragma(DigitalMars_intrinsic, "ind")
```

Comment:
How about `d.*` for intrinsics available on all compilers (e.g. `d.volatileLoad`) and `dmd.*`, `gdc.*`, `ldc.*`, etc. for non cross compiler portable ones?